src/qml/CategorySection.qml

```diff
@@ -22,6 +21,7 @@ import QtQuick 2.4
 import SystemSettings 1.0
 import SystemSettings.ListItems 1.0 as SettingsListItems
 import Ubuntu.Components 1.3
+import Ubuntu.Components.ListItems 1.3 as ListItem
```

Comment:
ListItems is [deprecated](https://api-docs.ubports.com/sdk/apps/qml/Ubuntu.Components.ListItems/Base.html#detailed-description). Please, don't use them in new code, intesad use ListItem

Reply:
OK, will change.

Reply:
Thank you

Reply:
I pushed a commit, to fix this, but I don't have the device with me right now, so you'd have to wait a couple of days, unless someone is able to test it before me.
Till then, please don't merge this (though I assume there's no risk of that happening in such a hurry :-) ).